certif/tests/acceptance/session-finalization_test.js

```diff
@@ -203,6 +203,13 @@ module('Acceptance | Session Finalization', function (hooks) {
             )
           )
           .exists();
+        assert
```

Comment:
On pourrait rajouter un not.to.exists lors du test avec le toggle à false
assert
          .dom(
            screen.getByText(
              'Pour que le signalement soit pris en compte, il est nécessaire d’utiliser la catégorie de signalement appropriée (exemples : C1, C2, etc).'
            )
          )
          .not.to.exists();